score/scrabble_score.rb

```diff
@@ -0,0 +1,81 @@
+module Scrabble
+
+  class Scrabble
+    # Set constants with general error message, point values of each letter, all letters
+    WORD_ERROR = "ERROR: Invalid input. Either your word has more that 7 letters or you entered something ridiculous. You only have 7 tiles, and Scrabble uses WORDS!"
```

Comment:
Extracting your errors messages is really smart, especially for future use. If you'd ever need to consider translating you app to another language, this will be super important.